packages/agoric-cli/src/commands/wallet.js

```diff
@@ -70,15 +86,27 @@ export const makeWalletCommand = async () => {
       normalizeAddress,
     )
     .requiredOption('--offer [filename]', 'path to file with prepared offer')
+    .option('--home [dir]', 'agd application home directory')
```

Comment:
I think these options can be put on the `'wallet'` command so they don't have to be repeated for each action

Reply:
I'm not having any luck hoisting the options like that.